src/transformers/onnx/features.py

```diff
@@ -192,6 +193,16 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=CamembertOnnxConfig,
         ),
+        "mobilebert": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "next-sentence-prediction",
```

Comment:
You'll need to add the mapping to the `AutoModelForNextSentencePrediction` to both `_TASKS_TO_AUTOMODELS` and `_TASKS_TO_TF_AUTOMODELS` attributes of the `FeaturesManager`. 

Could you also please add `next-sentence-prediction` to the BERT set of features?